src/display/metadata.js

```diff
@@ -12,21 +12,24 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-/* globals PDFJS, Document */
+/* globals Document */
 
 'use strict';
 
 (function (root, factory) {
   if (typeof define === 'function' && define.amd) {
-    define('pdfjs/display/metadata', ['exports', 'pdfjs/shared/util'], factory);
+    define('pdfjs/display/metadata', ['exports', 'pdfjs/shared/util',
+      'pdfjs/display/global'], factory);
   } else if (typeof exports !== 'undefined') {
-    factory(exports, require('../shared/util.js'));
+    factory(exports, require('../shared/util.js'), require('./global.js'));
   } else {
-    factory((root.pdfjsDisplayMetadata = {}), root.pdfjsSharedUtil);
+    factory((root.pdfjsDisplayMetadata = {}), root.pdfjsSharedUtil,
+      root.pdfjsDisplayGlobal);
   }
-}(this, function (exports, sharedUtil) {
+}(this, function (exports, sharedUtil, displayGlobal) {
 
 var error = sharedUtil.error;
+var PDFJS = displayGlobal.PDFJS;
 
 var Metadata = PDFJS.Metadata = (function MetadataClosure() {
```

Comment:
With this PR I want to keep already exposed API intact. However feature goal to move all PDFJS.xxxx to the single file (e.g. display/global.js) and use only display/api.js in the generic viewer.